src/wp-admin/contribute.php

```diff
@@ -0,0 +1,132 @@
+				<li><?php _e( '<strong>Translate</strong> WordPress into your local language' ); ?></li>
+				<li><?php _e( '<strong>Create</strong> and improve WordPress educational materials' ); ?></li>
+				<li><?php _e( '<strong>Promote</strong> the WordPress project to your community' ); ?></li>
+				<li><?php _e( '<strong>Take</strong> photos for or help curate the WordPress Photos Directory' ); ?></li>
+				<li><?php _e( '<strong>Organize</strong> or participate in local WordPress Meetups and WordCamps' ); ?></li>
+				<li><?php _e( '<strong>Lend</strong> your creative imagination to the WordPress UI designy' ); ?></li>
+				<li><?php _e( '<strong>Edit</strong> videos and add captions to WordPress.tv' ); ?></li>
+				<li><?php _e( '<strong>Explore</strong> ways to reduce the environmental impact of millions of websites around the world' ); ?></li>
+
+			</ul>
+		</div>
+		<div class="column">
+			<img src="data:image/svg+xml,%3Csvg width='436' height='300' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%231d35b4' /%3E%3C/svg%3E" alt="" />
+		</div>
+	</div>
+	<div class="about__section has-2-columns is-wider-right">
+		<div class="column">
+			<img src="data:image/svg+xml,%3Csvg width='436' height='300' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%231d35b4' /%3E%3C/svg%3E" alt="" />
+		</div>
+		<div class="column is-vertically-aligned-center">
+			<h3><?php _e( 'Code-based Contribution' ); ?></h3>
+			<p><?php _e( 'If you do code, or want to, you can contribute technically in numerous ways:' ); ?></p>
+			<ul>
+				<li><?php _e( '<strong>Find</strong> and report bugs in the WordPress core software' ); ?></li>
+				<li><?php _e( '<strong>Test</strong> new releases and proposed features for the block editor' ); ?></li>
+				<li><?php _e( '<strong>Write</strong> and submit patches to fix bugs or help build new features' ); ?></li>
+				<li><?php _e( '<strong>Contribute</strong> to the code, improve the UX, and test the WordPress app' ); ?></li>
+			</ul>
+			<p><?php _e( 'WordPress embraces new technologies, while being committed to backward compatibility. We use the following languages and libraries:' ); ?></p>
+			<ul>
+				<li><?php _e( 'For Core and Block Editor: HTML, CSS, PHP, SQL, JavaScript and React' ); ?></li>
+				<li><?php _e( 'For the WordPress app: Kotlin, Java, Swift, Objective-C, Vue, Python and TypeScript' ); ?></li>
+			</ul>
+		</div>
+	</div>
+
+	<div class="about__section">
+		<div class="column">
+			<h2><?php _e( 'Help Make WordPress Better' ); ?></h2>
+			<p><?php _e( 'Whether you speak at a local meetup, provide your perspective on a feature proposal, or submit a basic bug report, your contributions will make a difference.' ); ?></p>
+		</div>
+		<div class="column aligncenter">
+			<p><a href="https://make.wordpress.org/contribute/" target="_blank"><?php _e( 'Find your team &rarr;' ); ?></a></p>
```

Comment:
I don't think this page will be ready by the release date ([see the meta ticket](https://meta.trac.wordpress.org/ticket/7105)), maybe for now this can go to the Make WordPress homepage? If the page is launched before release, the link can be updated (since it's not a string to translate).

```suggestion
			<p><a href="https://make.wordpress.org/" target="_blank"><?php _e( 'Find your team &rarr;' ); ?></a></p>
```

Reply:
I believe we will be able to finish an MVP of the contributor tool to put on the /contribute page!